src/docs/sphinx/blueprint_mesh.rst

```diff
@@ -392,9 +392,228 @@ To conform, the ``state`` entry must be an *Object* and can have the following o
 Examples
 ~~~~~~~~~~~~~~~~~~~~~
 
-In C++, the ``conduit::blueprint::mesh::examples`` namespace and in Python the `conduit.blueprint.mesh.examples` module provide
-functions that generate example mesh blueprint data. For details on how to write these data sets to files, see the unit tests that exercise these examples in ``src/tests/blueprint/t_blueprint_mesh_examples.cpp``. This section outlines ``braid``, ``sprial``, and ``julia`` examples.
+The C++ ``conduit::blueprint::mesh::examples`` namespace and the Python `conduit.blueprint.mesh.examples` module provide
+functions that generate example mesh blueprint data. For details on how to write these data sets to files, see the unit
+tests that exercise these examples in ``src/tests/blueprint/t_blueprint_mesh_examples.cpp``. This section outlines the
+examples that demonstrate the most commonly used mesh schemas.
 
+gradient
```

Comment:
Thinking `gradient` might confuse folks (the dataset doesn't really contain a spatial gradient), should we just call it `basic`?

Reply:
Oops; I was considering doing a spatial gradient of the form `x+y+z` originally, but I changed it to IDs without modifying the name. I'll update this now!